### ISSUE TITLE
Not warning the usage of !important

### ISSUE THREAD

**Issue by skhaz** [stylelint/stylelint#5606]
> Clearly describe the bug

I thought that Stylelint was warn me about the use of `!imporntat`

> Which rule, if any, is the bug related to?

!imporntat

> What code is needed to reproduce the bug?

```css
export const Checkbox: AnyStyledComponent = styled(AntCheckbox)<CheckboxProps>`
  & {
    .ant-checkbox {
      top: auto;
    }

    * .ant-checkbox-inner {
      border-color: #7e77a1 !important;
    }
}
`
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "processors": ["stylelint-processor-styled-components"],
  "extends": ["stylelint-config-recommended", "stylelint-config-styled-components"]
}
```

> Which version of stylelint are you using?

`13.13.1`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

`"lint:css": "stylelint '**/*.tsx'",`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Yes, we are using `styled-components`

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

A warning about the usage of `!important`
